src/main/java/seedu/address/flashcard/Answer.java

```diff
@@ -29,4 +33,8 @@ public boolean checkAnswer(String userAnswer) {
     public String getAnswer() {
         return answer;
     }
+
+    public static boolean isValidAnswer(String test) {
```

Comment:
Would you mind adding documentation to this? So that others will understand what this method is doing.